main_bindata.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build bindata
+
+package main
+
+import (
+	"embed"
+	"io/fs"
+
+	"code.gitea.io/gitea/modules/migration"
+	"code.gitea.io/gitea/modules/options"
+	"code.gitea.io/gitea/modules/public"
+	"code.gitea.io/gitea/modules/templates"
+)
+
+//go:embed options public templates modules/migration/schemas
```

Comment:
```suggestion
//go:embed all:options all:public all:templates all:modules/migration/schemas
```

The `all:` prefix makes it embed files starting with `_` and `.` too. I ran into this issue in another app where a file named like `_.8538ed73.js` would not embed. Currently webpack outputs a file `-.8538ed73.js` so it would still embed, but I think it's better safe to add this now instead of be surprised in the future.